src/hooks/useRepayLoanHandler.tsx

```diff
@@ -65,19 +63,19 @@ export const useRepayLoanHandler = () => {
 
   useEffect(() => {
     if (gammaPool) {
-      getCfmm()
-        .then((res) => setCfmm(res))
+      getCfmmPoolAddr()
+        .then((res) => setCfmmPoolAddr(res))
         .catch((err) => console.log(err))
     }
```

Comment:
can we use the async/await pattern instead? is there a reason to use old JS promise pattern?

Reply:
updated to async/await